docs/cloud-native-security/cspm-get-started-azure.asciidoc

```diff
@@ -31,35 +31,44 @@ To set up CSPM for Azure, first add the CSPM integration, then enable cloud acco
 . From the Elastic Security *Get started* page, click *Add integrations*.
 . Search for `CSPM`, then click on the result.
 . Click *Add Cloud Security Posture Management (CSPM)*.
-. Under **Configure integration**, select **Azure**.
-. Give your integration a name that matches the purpose or team of the Azure subscription you want to monitor, for example, `azure-CSPM-1`.
+. Under **Configure integration**, select **Azure**, then select either **Azure Organization** or **Single Subscription**, depending on what resources you want to monitor. 
+. Give your integration a name that matches the purpose or team of the Azure resources you want to monitor, for example, `azure-CSPM-dev-1`.
 
 [discrete]
 [[cspm-set-up-cloud-access-section-azure]]
 === Set up cloud account access
 
-NOTE: To set up CSPM for an Azure subscription, you will need admin privileges for that subscription.
+NOTE: To set up CSPM for an Azure organization or subscription, you will need admin privileges for that organization or subscription.
 
 For most users, the simplest option is to use an Azure Resource Manager (ARM) template to automatically provision the necessary resources and permissions in Azure. If you prefer a more hands-on approach or require a specific configuration not supported by the ARM template, you can use one of the manual setup options described below.
 
 [discrete]
 [[cspm-set-up-ARM]]
 == ARM template setup (recommended)
 
+NOTE: If you are deploying to an Azure organization, you need the following permissions: `Microsoft.Resources/deployments/*`, `Microsoft.Authorization/roleAssignments/write`. You also need to https://learn.microsoft.com/en-us/azure/role-based-access-control/elevate-access-global-admin[Elevate access to manage all Azure subscriptions and management groups].
+
 . Under *Setup Access*, select *ARM Template*.
 . Under **Where to add this integration**:
 .. Select **New Hosts**.
-.. Name the {agent} policy. Use a name that matches the purpose or team of the cloud account or accounts you want to monitor. For example, `azure-dev-policy`. Click **Save and continue**. The *ARM Template deployment* window appears.
+.. Name the {agent} policy. Use a name that matches the resources you want to monitor. For example, `azure-dev-policy`. Click **Save and continue**. The *ARM Template deployment* window appears.
 .. In a new tab, log in to the Azure portal, then return to {kib} and click **Launch ARM Template**. This will open the ARM template in Azure.
+.. If you are deploying to an Azure organization, select the management group you want to monitor from the drop-down menu. Next, enter the subscription ID of the subscription where you want to deploy the VM that will scan your resources.
```

Comment:
A link to the subscriptions page might be helpful: https://portal.azure.com/#view/Microsoft_Azure_Billing/SubscriptionsBladeV2

Not sure if there is any risk for the link to stop working in the future.

Reply:
Thanks Orestis, we usually try to avoid linking to another company's docs for exactly the reason you mentioned. I'm hoping most people will know how to find the subscription ID, or can figure it out with a quick google.